ansible-ipi-install/inventory/hosts.sample

```diff
@@ -45,6 +45,10 @@ build=""
 # Provisioning IP address (default value)
 prov_ip=172.22.0.3
 
+# (Optional) Provisioning network configuration overrides (4.4+)
+#prov_bootstrap_ip=172.22.0.2
+#prov_dhcp_range="172.22.0.10,172.22.0.100"
+
```

Comment:
I don't think its this simple because we need to reflect that this could be an IPv6 bootstrap IP and range. I'll think about this more tomorrow

Reply:
I would think that as long as the `install-config.yml` is already able to accept IPv4 or IPv6, these values should be user-definable here, but I could be overlooking something for sure.

Reply:
They do, I just mean that I want to think about what the example above should look like to better reflect that it could be either or.